app/javascript/react/screens/App/Overview/screens/PlanWizard/components/PlanWizardInstancePropertiesStep/components/PlanWizardInstancePropertiesStepTable.js

```diff
@@ -105,7 +105,20 @@ class PlanWizardInstancePropertiesStepTable extends React.Component {
       const clusterId = additionalData.rowData.ems_cluster_id;
       const tenantId = destinationTenantIdsBySourceClusterId[clusterId];
       const tenant = tenantId && tenantsWithAttributesById[tenantId];
-      const options = tenant ? tenant[optionsAttribute] : [];
+      const allFitForVM = () => {
+        const { bestFitFlavors } = this.props;
+        const flavorsSlugPrefix = 'flavors/';
+        const allFits = bestFitFlavors.find(flavor => flavor.source_href_slug === `vms/${additionalData.rowData.id}`)
+          .all_fit;
```

Comment:
In theory, the `bestFitFlavors` collection should always contain an entry for each VM, but if something strange were to happen and it's not there, `[].find()` will return `undefined`